src/System.Security.Cryptography.Primitives/ref/System.Security.Cryptography.Primitives.cs

```diff
@@ -209,9 +209,9 @@ public abstract partial class SymmetricAlgorithm : System.IDisposable
     {
         protected int BlockSizeValue;
         protected int FeedbackSizeValue;
-        protected byte[] IVValue;
+        protected byte[]? IVValue;
         protected int KeySizeValue;
-        protected byte[] KeyValue;
+        protected byte[]? KeyValue;
```

Comment:
Can't SymmetricAlgorithm.Create(string) below return null?

Reply:
it can, misunderstood @bartonjs's comment